pkg/services/rediscache/bind.go

```diff
@@ -33,9 +36,18 @@ func (s *serviceManager) GetCredentials(
 	if err := service.GetStructFromMap(instance.SecureDetails, &sdt); err != nil {
 		return nil, err
 	}
+
+	uriStringTemplate := "redis://:%s@%s:%d"
```

Comment:
Let's put this in-line in the one place it's used. If this was copy/paste from another service, it was done like this in one or two places because of conditional tweaks to the template, but nothing like that is applicable here, so imo, this be more readable if it's in-line.